packages/uni-resolver-registrar-api/src/api-functions.ts

```diff
@@ -167,7 +168,13 @@ export function resolveDidEndpoint(router: Router, context: IRequiredContext, op
   })
 }
 
-export function deactivateDidEndpoint(router: Router, context: IRequiredContext, opts?: ISingleEndpointOpts) {
+/**
+ * @deprecated
+ * @param router
+ * @param context
+ * @param opts
+ */
+export function deleteDidEndpoint(router: Router, context: IRequiredContext, opts?: ISingleEndpointOpts) {
```

Comment:
Should this endpoint be removed? It seems that we ended up with duplicated code as `deleteDidEndpoint` and `deactivateDidEndpoint` receive the same parameters and call the same function.

Reply:
No, we should create a ticket to make it configurable. I want to also have endpoints that conform to proper REST standards, as the DIF/W3C VC APIs typically are terrible in design

Reply:
Also we have actual customers using it (for instance GS1)